### PR TITLE
Bugfix/fix state of currency selector component (#2795)

### PR DESCRIPTION
**Pull request by SirZemar**
* Fix state of currency selector component

### CODE REVIEW

File: libs/ui/src/lib/currency-selector/currency-selector.component.html [hunk 1]
```diff
@@ -8,12 +8,11 @@
 
 <mat-autocomplete
   #currencyAutocomplete="matAutocomplete"
-  [displayWith]="displayFn"
   (optionSelected)="onUpdateCurrency($event)"
 >
   @for (currencyItem of filteredCurrencies; track currencyItem) {
```
Comment (dtslvr) [inline libs/ui/src/lib/currency-selector/currency-selector.component.html:13]: `@for (currency of filteredCurrencies; track currency) {`

File: libs/ui/src/lib/currency-selector/currency-selector.component.html [hunk 2]
```diff
@@ -8,12 +8,11 @@
...
     <mat-option class="line-height-1" [value]="currencyItem">
-      {{ currencyItem.label }}
+      {{ currencyItem }}
```
Comment (dtslvr) [inline libs/ui/src/lib/currency-selector/currency-selector.component.html:15]: `{{ currency }}`

**Review by dtslvr — CHANGES_REQUESTED** [review]
Nice. Looks a bit cleaner now 👍🏻 There is one suggestion for refactoring in the template.

**Comment by dtslvr** [comment]
@SirZemar have you seen my feedback?

**Comment by SirZemar** [comment]
I will take care of it, thank you @dtslvr for calling out to my attention.

**Review by dtslvr — APPROVED** [review]
Thank you for the changes, @SirZemar!

While testing I found an issue in the account dialog and fixed it via https://github.com/ghostfolio/ghostfolio/pull/3429/commits/aca6ccdb60c95f649fd70f46d01f29d62ce3815d. Ready to merge now.

**Pull request merged**